src/app.js

```diff
@@ -54,66 +59,49 @@ class App extends Component {
   }
   
   render() {
-    
-    const listBox = {
-      padding: '5px',
-      borderTop: '1px solid black',
-      borderLeft: '1px solid black',
-      borderRight: '1px solid black',
+    const forest = {
```

Comment:
Move these style object literal declarations outside of `render` method to avoid having to allocate everytime `render` is called. You can move these outside of the component itself into the module.